src/components/__tests__/sliding-panel.spec.tsx

```diff
@@ -16,18 +16,45 @@ describe('SlidingPanel component', () => {
         </SlidingPanel>
       </>
     );
+    fireEvent.click(screen.getByText('Sliding panel content'));
+    await waitFor(() => sleep(200));
```

Comment:
Good point. I had previously tried to pass some kind of condition that would make sense in the waitFor, but since nothing changes at all (that's the whole point), I defaulted to just sleeping, but the waitFor is not really useful then 😅